src/main/java/frc/robot/Constants.java

```diff
@@ -65,11 +66,26 @@ public static class Drive {
     /** Empty translation to prevent creating 2 Translation2ds every time the drive train stops. */
     public static Translation2d EMPTY_TRANSLATION = new Translation2d();
 
-    /** The max speed the robot can go */
+    /** The max speed the robot can go in m/s */
     public static double MAX_SPEED = 1;
 
     /** The max speed the robot can rotate */
     public static double MAX_ANGULAR_SPEED = Math.PI / 2;
+
+    /** The distance from the center of the robot to any of the swerve modules. */
+    public static double DISTANCE_ROBOT_CENTER_TO_SWERVE_MODULE = 0.3;
+
+    /**
+     * The Translation Drive PID for the robot. Since we aren't using drive PID this year, the
+     * values are set to the default.
+     */
+    public static PIDConstants TRANSLATION_DRIVE_PID = new PIDConstants(1.0, 0.0, 0.0);
+
+    /**
+     * The Rotation Drive PID for the robot. Since we aren't using drive PID this year, the values
```

Comment:
Delete the "Since we aren't using drive PID..." sentence. We will eventually need to tune these values.